files/default/neutron-evacuate-lbaasv2-agent.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/python
+#
+# Copyright 2017, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import importlib
+import socket
+import sqlalchemy
+from oslo_utils import timeutils
+from oslo_config import cfg
+from oslo_log import log as logging
+
+LOG = logging.getLogger(__name__)
+
+# Try to fall back loading the neutron-ha-tool from an absolute path.
+# As on the installed system we're installing neutron-ha-tool without
+# the ".py" extension, which causes importlib.import_module to fail.
+try:
+    hatool = importlib.import_module("neutron-ha-tool")
+except ImportError:
+    import imp
+    import os
+    dirname = os.path.dirname(os.path.abspath(__file__))
+    hatool = imp.load_source(
```

Comment:
It will fail like this:

```
Traceback (most recent call last):
  File "/usr/bin/neutron-evacuate-lbaasv2-agent", line 37, in <module>
    '', os.path.join(dirname, '/usr/bin/neutron-ha-tool'))
IOError: [Errno 2] No such file or directory
```

It's not the most user friendly kind of failure, but somehow I think it's good enough for a tool like this.